modules/dnn/src/layers/is_inf_layer.cpp

```diff
@@ -0,0 +1,86 @@
+// This file is part of OpenCV project.
+// It is subject to the license terms in the LICENSE file found in the top-level directory
+// of this distribution and at http://opencv.org/license.html.
+// Copyright (C) 2025, BigVision LLC, all rights reserved.
+// Third party copyrights are property of their respective owners.
+
+#include "../precomp.hpp"
+#include "layers_common.hpp"
+#include "opencv2/core/fast_math.hpp"  // for cvIsInf
+
+namespace cv {
+namespace dnn {
+
+template <typename T>
+static inline void computeIsInfMask(const T* src, uchar* dst, const size_t count, const bool detectPositive, const bool detectNegative)
+{
+    for (size_t i = 0; i < count; ++i)
```

Comment:
it would be nice to make this loop parallel just to eliminate possible bottlenecks when everything else in the graph is parallel